src/seedu/addressbook/data/person/Person.java

```diff
@@ -62,6 +62,9 @@ public Address getAddress() {
         return new HashSet<>(tags);
     }
 
+    @Override
+    public changeName(String newName){this.name = newName;}
```

Comment:
Person class implements ReadOnlyPerson class. The method that you have created changeName infringes the property of ReadOnlyPerson (to be read only and not modified).